src/net/rpc.rs

```diff
@@ -417,7 +417,12 @@ impl RPCPoxInfoData {
             .ok_or_else(|| net_error::DBError(db_error::Overflow))?
             as u64;
 
-        let cur_cycle_pox_active = sortdb.is_pox_active(burnchain, &burnchain_tip)?;
+        let cur_cycle_pox_active = sortdb
+            .is_pox_active(burnchain, &burnchain_tip)
+            .unwrap_or_else(|e| {
+                error!("Error in /v2/pox while determining if PoX is active: {}", e);
+                false
```

Comment:
Do we want to make the `is_pox_active` an option here...? returning `false` seems a bit misleading to me, whereas `None` more explicitly shows the lack of data

Reply:
I imagine (after the node has boostrapped enough burnchain blocks), that this value is only ever true/false -- and that a None/error value is a bug.

Reply:
That is a fair point. However, the false case for `is_pox_active` as is seems overloaded - it's used both to signal the node is bootstrapping & that the network voted to reject pox for that cycle.